front/src/utils/date.ts

```diff
@@ -162,3 +164,9 @@ export function convertIsoUtcToLocalTime(isoUtcString: string): string {
 export function addDurationToIsoDate(startTime: string, duration: string) {
   return dayjs(startTime).add(ISO8601Duration2sec(duration), 'second').format();
 }
+
+export function transformDateWithoutYear(dateTime: string) {
```

Comment:
> Returning an empty string when the input format is invalid is unpredictable and must be documented

Means that the function **MUST** be documented and should not have this unpredictable behavior.

> [!NOTE]
> If you take a `Date` as input you don't have to handle the case where the string is not a formated date.